.gitignore

```diff
@@ -20,6 +20,9 @@ docs/_build
 docs/recipes/data
 docs/recipes/.ipynb_checkpoints
 
+# editor files
+.ipynb_checkpoints
+
```

Comment:
I would not add `.ipynb_checkpoints` at the root level, you should only have and run notebooks at the `docs/recipes` folder.

Reply:
I did only run notebooks at the docs/recipes folder, but I opened a file in the jupyter lab editor and it created the directory (it would have been staged for commit if I had done `git add .`). I added it becase I suppose we don't want hidden checkpoints folders in the code structrure in general but I can remove it in a new commit.

Reply:
ok, in that case we will remove the line above `docs/recipes/.ipynb_checkpoints`